README.md

```diff
@@ -97,14 +97,37 @@ $ balena run -it -v ${appid}_resin-data:/mnt alpine \
 
 ## Networking
 ### Shared physical device
-First, create a bridge, and assign the physical interface as a slave. This can be done on the host OS using nmcli:
+
+Check your network before you wreck your network:
+
+Find the ethernet name for example something like this eno1 or enp89s0 
+`` 
+$ ip a
```

Comment:
This is a good suggestion, I can see how it was confusing not understanding that `eno1` later is machine specific